docs/user-guide/administration/manage-etcd.md

```diff
@@ -0,0 +1,228 @@
+etcd-0   1/1     Running   0          64s
+etcd-1   1/1     Running   0          65s
+etcd-2   1/1     Running   0          72s
+```
+</details>
+
+The etcd [initialClusterState](https://etcd.io/docs/v3.5/op-guide/configuration/) parameter specifies the initial state of the etcd cluster when starting etcd nodes. It is important for determining how the node will join the cluster. The parameter can take the following two values:
+
+- **new**: This value indicates that the etcd cluster is new. All nodes will start as part of a new cluster, and any previous state will not be used.
+- **existing**: This value indicates that the node will join an already existing etcd cluster. In this case, you must ensure that the initialCluster parameter is configured with the information of all nodes in the current cluster.
+
+After the etcd cluster is running, we need to set the initialClusterState parameter to **existing**:
+
+```bash
+helm upgrade --install etcd \
+  oci://registry-1.docker.io/bitnamicharts/etcd \
+  --version 10.2.12 \
+  --set initialClusterState="existing" \
+  --set removeMemberOnContainerTermination=false \
+  --set replicaCount=3 \
+  --set auth.rbac.create=false \
+  --set auth.rbac.token.enabled=false \
+  --create-namespace \
+  -n etcd-cluster
+```
+
+Wait for etcd cluster to be running, use the following command to check the health status of etcd cluster:
+
+```bash
+kubectl -n etcd-cluster \
+  exec etcd-0 -- etcdctl \
+  --endpoints etcd-0.etcd-headless.etcd-cluster:2379,etcd-1.etcd-headless.etcd-cluster:2379,etcd-2.etcd-headless.etcd-cluster:2379 \
+  endpoint status -w table
+```
+
+<details>
+  <summary>Expected Output</summary>
+```bash
++----------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+|                ENDPOINT                |        ID        | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS |
++----------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+| etcd-0.etcd-headless.etcd-cluster:2379 | 680910587385ae31 |  3.5.15 |   20 kB |     false |      false |         4 |      73991 |              73991 |        |
+| etcd-1.etcd-headless.etcd-cluster:2379 | d6980d56f5e3d817 |  3.5.15 |   20 kB |     false |      false |         4 |      73991 |              73991 |        |
+| etcd-2.etcd-headless.etcd-cluster:2379 | 12664fc67659db0a |  3.5.15 |   20 kB |      true |      false |         4 |      73991 |              73991 |        |
++----------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+```
+</details>
+
+## Backup
+
+Add the following configuration and name it `etcd-backup.yaml` file, Note that you need to modify **existingClaim** to your nfs pvc name:
```

Comment:
what are nfs and pvc?
If it is an abbreviation, use format like `Nxx Fxx Sxx (NFS)` when writing it for the first time